mpf/platforms/system11.py

```diff
@@ -60,25 +62,26 @@ def __init__(self, machine: MachineController) -> None:
     def stop(self):
         """Stop the overlay. Nothing to do here because stop is also called on parent platform."""
 
+    @property
+    def a_side_active(self):
+        """Return if A side cannot be switched off right away."""
+        return self.drivers_holding_a_side or self.a_side_done_time + self.debounce_secs > self.machine.clock.get_time()
+
     @property
     def a_side_busy(self):
-        """Return if A side cannot be switches off right away."""
-        return self.drivers_holding_a_side or self.a_side_done_time > self.machine.clock.get_time() or self.a_side_queue
+        """Return if A side cannot be switched off right away."""
+        return self.a_side_active or self.a_side_queue
```

Comment:
This block (and the C side) originally duplicated the check in both `a_side_active` and `a_side_busy`, so I cleaned the latter up to just check the former.